lib/mayaUsd/ufe/UsdAttributeHolder.cpp

```diff
@@ -54,10 +51,10 @@ bool setUsdAttrMetadata(
     }
 
     // If attribute is locked don't allow setting Metadata.
-    enforceAttributeEditAllowed(attr);
+    MAYAUSD_NS_DEF::ufe::enforceAttributeEditAllowed(attr);
```

Comment:
In actual code we don't use the macro. This should just be "MayaUsd".

Reply:
There are some other places in the code that are also wrong, so I'm sorry if you copy-pasted one of them. These need to be cleaned up (in a separate PR).